clojure-ts-mode.el

```diff
@@ -1872,6 +1872,31 @@ functional literal node."
       (clojure-ts--skip-first-child threading-sexp)
       (not (treesit-end-of-thing 'sexp 2 'restricted)))))
 
+(defun clojure-ts--raise-sexp ()
+  "Raise current sexp one level higher up the tree.
+
+The built-in `raise-sexp' function doesn't work well with a few Clojure
+nodes (function literals, expressions with metadata etc.), it loses some
+parenthesis."
+  (when-let* ((sexp-node (treesit-thing-at (point) 'sexp))
+              (beg (thread-first sexp-node
+                                 (clojure-ts--node-start-skip-metadata)
+                                 (copy-marker)))
+              (end (thread-first sexp-node
+                                 (treesit-node-end)
+                                 (copy-marker))))
+    (when-let* ((parent (treesit-node-parent sexp-node))
```

Comment:
Why add a second `when-let*` here?

Reply:
I think I had an idea initially to make it interactive and signal a different errors, but then I changed my mind. I'll fix that shortly. Thanks!